assets/questions/i18n.json

```diff
@@ -1,37 +1,124 @@
 {
-    "language": "python",
-    "stages": [{
-        "instructions": "Welcome to this programming exercise. Implement the abbreviate function.",
-        "data": [{
+  "interview_question": {
+    "title": "Internationalization (i18n)",
+    "prompts": [
+      {
+        "instructions": [
+          "Welcome to this programming exercise.",
+          "You will complete this exercise by editing the code in the sample.py file using your favorite editor. Every time you save the file, I will evaluate your code and let you know about next steps.",
+          "Your first task is to implement the abbreviate function. It takes a string as input and returns an abbreviation of the string of the form <first character><length of the middle of the string><last character>. For example, \"internationalization\" should be abbreviated as \"i18n\"."
+        ],
+        "prerequisite_skills": ["Arrays", "Strings", "String Manipulation"],
+        "acquired_skills": ["String Manipulation"],
+        "input_function": null,
+        "output_function": null,
+        "main_function": "sample.abbreviate",
+        "correctness_test_data": [
+          {
             "input": "internationalization",
             "output": "i18n"
-        }, {
+          },
+          {
             "input": "monkey",
             "output": "m4y"
-        }, {
+          },
+          {
             "input": "friendship",
             "output": "f8p"
-        }],
-        "functionName": "abbreviate"
-    }, {
-        "instructions": "Make sure your code handles short strings correctly.",
-        "data": [{
+          }
+        ],
+        "buggy_outputs": [
+          {
+            "buggy_function": "forgetLastLetter",
+            "messages": [
+              "It looks like your output (%s) doesn't match our expected output (%s).",
+              "You seem to be dropping the last character of the string when you're abbreviating.",
+              "Make sure to add the last character of the string back on when you've abbreviated."
+            ]
+          }
+        ],
+        "performance_test_data": {
+        }
+      },
+      {
+        "instructions": [
+          "You need to make sure that your code handles short strings correctly.",
```

Comment:
Fine for now but I think we may want to rephrase this in the future to something like:
"It's always important to think about the edge cases for your code. In this case, short strings are not abbreviated by this strategy. Make sure your code doesn't try to abbreviate when it's not necessary."